uav_pid_ctrl1:
  rate: 100
  state_topic: odom
  ref_topic: cmd_pos
  command_topic: cmd_vel
  mass: 1.027
  inertia: {ixx: 2.3951e-1, iyy: 2.3951e-1, izz: 3.2347e-1}
  position_pid: {p: 1, i: 0, d: 1}
  rpy_pid: {p: 0.1, i: 0, d: 0.1}